test-framework/core/src/main/java/org/keycloak/testframework/server/EmbeddedKeycloakServer.java

```diff
@@ -18,6 +23,27 @@ public void start(KeycloakServerConfigBuilder keycloakServerConfigBuilder) {
             builder.addDependency(dependency.getGroupId(), dependency.getArtifactId(), "");
         }
 
+        Set<Path> configFiles = keycloakServerConfigBuilder.toConfigFiles();
+        if (!configFiles.isEmpty()) {
+            Path homeDir = Platform.getPlatform().getTmpDirectory().toPath();
+            Path conf = homeDir.resolve("conf");
+
+            if (!conf.toFile().exists()) {
+                conf.toFile().mkdirs();
+            }
+
+            for (Path configFile : keycloakServerConfigBuilder.toConfigFiles()) {
```

Comment:
@mhajas here you can reuse the configFiles parameter instead of using builder again

Reply:
done